server/src/infra/tcp_server.rs

```diff
@@ -76,8 +70,16 @@ fn http_config<Backend>(
     )
     // Serve the /pkg path with the compiled WASM app.
     .service(Files::new("/pkg", "./app/pkg"))
-    // Default to serve index.html for unknown routes, to support routing.
-    .service(web::scope("/").route("/.*", web::get().to(index)));
+    // Serve static files
+    .service(Files::new("/static", "./app/static"))
+    // Serve static fonts
+    .service(Files::new("/static/fonts", "./app/static/fonts"))
+    // Serve the index
+    .service(
+        web::scope("/")
```

Comment:
We need to serve every 404 with the index, for routing purposes.
If the user loads the page "http://server.tld/user/bob" we need to serve the index, and the SPA logic will route to the correct page.

Reply:
I was wondering why that wildcard route was there. I was also wondering why every incorrect `/...` request failed. 🤦🏽‍♂️

Reply:
Sorry, last nit: can you add back the comment?
// Default to serve index.html for unknown routes, to support routing.

Reply:
Done!